dashboard/config/routes.rb

```diff
@@ -56,6 +56,7 @@
   get 'maker/application_status', to: 'maker#application_status'
   post 'maker/override', to: 'maker#override'
   get 'maker/login_code', to: 'maker#login_code'
+  get 'maker/display_code', to: 'maker#display_code'
```

Comment:
since this is google-specific, i think this route should make that clear (maybe `/maker/display_google_oauth_code` or something similar? i don't feel passionate about a specific name)